.circleci/push_doc.sh

```diff
@@ -73,10 +73,15 @@ git config user.name $USERNAME
 git config push.default matching
 git add -f $dir/
 git commit -m "$MSG" $dir
-if [ "$CIRCLE_BRANCH" = "main" ]
+
+# See if the JS dir changed
+modified_js = `git status -s -uno js | wc -l`
+if [ "$CIRCLE_BRANCH" = "main"  && $modified_js -ne 0 ]
 then
     git add -f js/
-    git commit -m "$MSG" js/
+    git commit -m "JS dir update: $MSG" js/
```

Comment:
could also add --allow-empty here ;) but this works as well.

Reply:
Turns out is doesn't.... not quite sure why `bash` is saying that `modified_js` is not found. Expect another PR shortly.